assets/models/system/openai-clip-image-text-embeddings-vit-large-patch14-336/spec.yaml

```diff
@@ -0,0 +1,80 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: OpenAI-CLIP-Image-Text-Embeddings-ViT-Large-Patch14-336
+path: ./
+properties:
+    inference-min-sku-spec: 2|0|7|14
+    inference-recommended-sku: Standard_DS2_v2, Standard_D2a_v4, Standard_D2as_v4, Standard_DS3_v2,
+        Standard_D4a_v4, Standard_D4as_v4, Standard_DS4_v2, Standard_D8a_v4, Standard_D8as_v4,
+        Standard_DS5_v2, Standard_D16a_v4, Standard_D16as_v4, Standard_D32a_v4, Standard_D32as_v4,
+        Standard_D48a_v4, Standard_D48as_v4, Standard_D64a_v4, Standard_D64as_v4, Standard_D96a_v4,
+        Standard_D96as_v4, Standard_F4s_v2, Standard_FX4mds, Standard_F8s_v2, Standard_FX12mds,
+        Standard_F16s_v2, Standard_F32s_v2, Standard_F48s_v2, Standard_F64s_v2, Standard_F72s_v2,
+        Standard_FX24mds, Standard_FX36mds, Standard_FX48mds, Standard_E2s_v3, Standard_E4s_v3,
+        Standard_E8s_v3, Standard_E16s_v3, Standard_E32s_v3, Standard_E48s_v3, Standard_E64s_v3,
+        Standard_NC4as_T4_v3, Standard_NC6s_v3, Standard_NC8as_T4_v3, Standard_NC12s_v3,
+        Standard_NC16as_T4_v3, Standard_NC24s_v3, Standard_NC64as_T4_v3, Standard_NC24ads_A100_v4,
+        Standard_NC48ads_A100_v4, Standard_NC96ads_A100_v4, Standard_ND96asr_v4, Standard_ND96amsr_A100_v4,
+        Standard_ND40rs_v2
+tags:
+    Preview: ''
+    huggingface_model_id: openai/clip-vit-large-patch14-336
+    SharedComputeCapacityEnabled: ''
+    author: openai
+    license: mit
+    task: embeddings
+    inference_compute_allow_list:
+        [
+            Standard_DS2_v2,
```

Comment:
Just making sure: did you test CLIP-L@336 on Ds2_v2?

Reply:
yes, I used DS2_v2 as the inference compute for the publishing pipeline